src/index.html

```diff
@@ -3,10 +3,497 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.googleapis.com"
+    >
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+    >
+    <link 
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" 
+      rel="stylesheet"
+    >
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link 
+      rel="stylesheet" 
+      href="./styles/main.scss"
+    >
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a 
+        href="index.html"
+        class="header__logo-link"
+      >
+        <img 
+          src="/images/logo.png" 
+          alt="logo"
+          class="header__logo-img"
+        >
+      </a>
+      <nav>
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a 
+              href="#" 
+              class="header__nav-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          
+          <li class="header__nav-item">
+            <a href="#" class="header__nav-link ">Samsung</a>
+          </li>
+          
+          <li class="header__nav-item">
+            <a href="#" class="header__nav-link ">Smartphones</a>
+          </li>
+          
+          <li class="header__nav-item">
+            <a 
+              href="#" 
+              class="header__nav-link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          
+          <li class="header__nav-item">
+            <a href="#" class="header__nav-link ">Gadgets</a>
+          </li>
+          
+          <li class="header__nav-item">
+            <a href="#" class="header__nav-link ">Tablets</a>
+          </li>
+          
+          <li class="header__nav-item">
+            <a href="#" class="header__nav-link ">Photo</a>
+          </li>
+          
+          <li class="header__nav-item">
+            <a href="#" class="header__nav-link ">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    
+    <main class="main">
+      <div class="main__container">
+        <div 
+          class="card main__card-position"
+          data-qa="card"
+        >
+
+          <img 
+            src="./images/imac.jpeg" 
+            alt="images"
+            class="card__image"
+          >
+        
+          <h2 class="card__product-name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          
+          <p class="card__product-code">
+            Product code: 195434
+          </p>
+          
+          <div class="card__product-rate">
+            <div class="stars">
```

Comment:
use logic from [stars task](https://github.com/mate-academy/layout_stars)

Reply:
not fixed, check task description

Reply:
Can you please advise what is wrong with block stars ? Schould I rename it for follow BEM naming or ...?  I used loop for@ , is it wrong?